tests/functional/test_tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = py26,py27,pypy,py32,py33,py34,pypy3
+skipsdist = true
+toxworkdir = {env:TMPDIR}
+
+[testenv]
+deps =
+    nameless
+commands =
+    nameless
+
+[testenv:py26]
+basepython = {env:TOXPYTHON:python2.6}
+
+[testenv:py27]
+basepython = {env:TOXPYTHON:python2.7}
+
+[testenv:py32]
+basepython = {env:TOXPYTHON:python3.2}
+
+[testenv:py33]
+basepython = {env:TOXPYTHON:python3.3}
+
+[testenv:py34]
+basepython = {env:TOXPYTHON:python3.4}
```

Comment:
Yes indeed you are correct, they are not there cause I didn't add PyPy into the AppVeyor test grid yet. It's on my todo list